omod/src/main/resources/org/openmrs/module/fhir2/narratives/Person.html

```diff
@@ -31,7 +31,7 @@
         <th:block th:if="resource.photo">
             <tr th:if="${resource.photo.url}">
                 <td>Photo:</td>
-                <td><img src="${resource.photo.url}" /></td>
```

Comment:
@ShivanshRakesh  Thanks for spotting this change.
The pr looks good to me. @ibacher   Hi ian,how does this entire pr look to you cc @dkayiwa

Reply:
Thanks @HerbertYiga! :)